documentation/modules/proc-kafka-connect-mounting-volumes.adoc

```diff
@@ -43,42 +55,48 @@ metadata:
 spec:
   # ...
   config:
-    config.providers: file
-    config.providers.file.class: org.apache.kafka.common.config.provider.FileConfigProvider
+    config.providers: file <1>
```

Comment:
is there a kind of list of supported config providers somewhere @tombentley ?

Reply:
So I think that we should explain that in the examples as you highlight that `config.providers` defines an alias and you have to use that alias in the `config.proviers.${alias}.class` parameter. Tbh it was not clear to me.

Reply:
@PaulRMellor ^^

Reply:
@ppatierno - how about:

> <1> The alias for the configuration provider, which is used to define other configuration properties. 
> Use a comma-separated list if you want to add more than one provider.
> <2> The `FileConfigProvider` is the configuration provider that provides values from properties files.
> The property uses the alias from `config.providers`, taking the form `config.providers.${alias}.class`.